cmd/plugin/cli/common.go

```diff
@@ -173,3 +174,27 @@ func GetBoolIfConfigSet(key string) *bool {
 	}
 	return nil
 }
+
+func storageosCommand(cmd *cobra.Command, args []string) []string {
+	commands := []string{"storageos"}
+	if cmd.HasParent() && cmd.Parent().Use != "kubectl-storageos" {
+		commands = append(commands, cmd.Parent().Use)
+	}
+	commands = append(commands, cmd.Use)
+
+	return append(commands, parseArgs(args)...)
+}
+
+func parseArgs(args []string) []string {
+	parsedArgs := make([]string, 0)
+	for _, individualArg := range args {
+		if strings.Contains(individualArg, "=") {
+			argsSeparated := strings.Split(strings.ReplaceAll(individualArg, "=", " "), " ")
```

Comment:
Does it make sense to use some special character like ¬, ¦, € instead of space? I think this code should be a source of some strange bugs. Or cut by the first index of `=`?

Reply:
Good idea, will use a special char instead of space.